packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1121,7 +1121,7 @@ oie.idp.authenticator.description = Redirect to verify with {0}.
 # {0} refers to the display name of the authenticator
 oie.idp.enroll.title = Set up {0}
 # {0} refers to the display name of the authenticator
-oie.idp.enroll.description = Clicking below will redirect to enrollment in {0}
+oie.idp.enroll.description = You will be redirected to enroll with {0}
```

Comment:
I believe this should be "... to enroll **in** {0}". Feel free to check with UX or your doc writer

Reply:
@robertmilkovich-okta @vincentvoong-okta Could you help me with the wording to reflect auto-redirect as well? thanks!

Reply:
thanks @robertmilkovich-okta for the help on wording. I updated the wording as per his feedback.

Reply:
The new sentence does not work in all cases. We show the same sentence whether `features.skipIdpFactorVerificationBtn` is `true` or `false`. The way this is worded now ONLY works when the feature is true. If the feature is `false` a user now will see this sentence "Redirecting to..." and below a button, so there is no redirect until the button is clicked. Please change it to `You will be redirected to enroll in {0}` or check with your doc writer and explain both flows, since the sentence applies to both. 

Another option is to have 2 different labels, one for when the feature is on, and another one when it is off

Reply:
@mauriciocastillosilva-okta doc writer provided this text applicable to both cases.

@robertmilkovich-okta any insight on Mau's comment above? thanks!

Reply:
It looks like you have to have two sentences, one for using with the button, and one for when it’s automatic. So, use these phrases:
With clickable button:
“You will now be redirected to enroll in the IdP Authenticator”
For automatic redirecting:
“Redirecting you to enroll in the IdP Authenticator”